tests/e2e/basic/configuration_tests/nat_mode/test_enterprise_security_modes_configuration.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+import allure
+
+pytestmark = [pytest.mark.setup, pytest.mark.nat, pytest.mark.sanity, pytest.mark.enterprise, pytest.mark.usefixtures("setup_test_run")]
+
+setup_params_enterprise = {
+    "mode": "NAT",
+    "ssid_modes": {
+        "wpa_enterprise": [
+            {"ssid_name": "ssid_wpa_eap_2g", "appliedRadios": ["is2dot4GHz"]},
+            {"ssid_name": "ssid_wpa_eap_5g", "appliedRadios": ["is5GHzU", "is5GHz", "is5GHzL"]}],
+        "wpa2_enterprise": [
+            {"ssid_name": "ssid_wpa2_eap_2g", "appliedRadios": ["is2dot4GHz"]},
+            {"ssid_name": "ssid_wpa2_eap_5g", "appliedRadios": ["is5GHzU", "is5GHz", "is5GHzL"]}],
+        "wpa3_enterprise": [
+            {"ssid_name": "ssid_wpa3_eap_2g", "appliedRadios": ["is2dot4GHz"]},
+            {"ssid_name": "ssid_wpa3_eap_5g", "appliedRadios": ["is5GHzU", "is5GHz", "is5GHzL"]}]},
+
+    "rf": {},
```

Comment:
we do have options to change rf configuration, whatever key valure pair we enter, it can be changed. but for every test case as of now, we didn't need any change in rf profile and thats why its taking default ones if it is empty.
i can add the logic for that as needed but since we are moving to ucentral so i havent spent much time on that, but framework allows us to do that